### DISCUSSION TITLE
saf-training/courses/beginner/12

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#31] [Ideas]
# saf-training/courses/beginner/12

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/beginner/12.html

**Comment by karikarshivani** [comment]
Include git push commands to save work

**Comment by rooster-1** [comment]
The instructional text for step 1 reads: "Read the control - when referencing the control SV-230324 from the stub profile that was generated with the stub generator...."

However the subsequent example uses control SV-258043 (which is among the controls listed when running the stub generator as of May 2025). Consider updating instructional text to read "SV-258043"
